model/src/main/kotlin/VulnerabilityReference.kt

```diff
@@ -51,14 +51,16 @@ data class VulnerabilityReference(
     val severity: String?
 ) {
     companion object {
+        private val SEVERITIES = listOf("NONE", "LOW", "MEDIUM", "HIGH", "CRITICAL")
```

Comment:
Would it make sense to use `enumValues<Cvss3Rating>().map { it.name }` instead? (Esp. if we'd know that VC is actually using CVSS3 names here.)